### PR TITLE
added Address Index API Swagger file

### PR DESCRIPTION
**Pull request by alangorton**

### CODE REVIEW

File: api-definitions/address-index-api.yaml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+      summary: Gets an address by UPRN.
+      description: Returns a single address, identified by its UPRN.
+
+      parameters:
+       - name: uprn
+         in: path
+         type: string
+         required: true
+         description: Specifies the address UPRN.
+
```
Comment (Mironor) [inline api-definitions/address-index-api.yaml]: Add format parameter

**Comment by Mironor** [comment]
LGTM

File: api-definitions/address-index-api.yaml [hunk 2]
```diff
@@ -0,0 +1,326 @@
+swagger: "2.0"
+
+info:
+  title: ONS Address Index API
+  description: ONS Address Index API
```
Comment (iRhysBradbury) [inline api-definitions/address-index-api.yaml]: Full stop, all descriptions in this file have a full stop

Comment (Mironor) [inline api-definitions/address-index-api.yaml]: Ah, the period...

File: api-definitions/address-index-api.yaml [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+
+       - name: format
+         in: query
+         type: string
+         required: false
+         description: Specifies the response address format, either paf or bs7666 (defaults to paf).
+
+      responses:
+        200:
+          description: OK.
```
Comment (iRhysBradbury) [inline api-definitions/address-index-api.yaml]: better desc's than rest standard please.

**Pull request merged**